### PR TITLE
LP-2915 Add calendar links

### PR DESCRIPTION
**Pull request by javeriaejaz-arbisoft**
[LP-2915](https://philanthropyu.atlassian.net/browse/LP-2915)

[Theme PR](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/###)

***

#### Description:
Add google & yahoo calendar links 
Add option to download calendar file

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3184/workflows/workflows_id/jobs/3184/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/tests/test_views.py (100%)
openedx/adg/lms/webinars/views.py (100%)
-------------
Total:   43 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by aightmunam — APPROVED** [review]
Looks good to me!

File: openedx/adg/lms/webinars/views.py [hunk 1]
```diff
@@ -50,6 +56,71 @@ def get_context_data(self, **kwargs):
...
+        event.add('summary', webinar.title)
+        event.add('description', webinar.description)
+        event.add('dtstart', webinar.start_time)
+        event.add('duration', webinar.duration)
+        event.add('location', webinar.meeting_link)
+        event.add('uid', uuid.uuid4())
+        cal.add_component(event)
+
+        response = HttpResponse(cal.to_ical(), content_type='text/calendar; charset=utf8')
+        response['Content-Disposition'] = 'attachment; filename=calendar.ics'
```
Comment (aightmunam) [inline openedx/adg/lms/webinars/views.py:124]: Wouldn't it be better to include title of the webinar in the downloaded filename?

**Review by faizan-philu — DISMISSED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3189/workflows/workflows_id/jobs/3189/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/tests/test_views.py (100%)
openedx/adg/lms/webinars/views.py (100%)
-------------
Total:   43 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3190/workflows/workflows_id/jobs/3190/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/tests/test_views.py (100%)
openedx/adg/lms/webinars/views.py (100%)
-------------
Total:   43 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by aightmunam — APPROVED** [review]

File: openedx/adg/lms/webinars/views.py [hunk 2]
```diff
@@ -50,6 +56,72 @@ def get_context_data(self, **kwargs):
...
+        Returns:
+            str: google link
+        """
+        base_url = 'https://www.google.com/calendar/event?'
+        end_time = self.object.start_time + self.object.duration
+        details = {
+            'action': 'TEMPLATE',
+            'text': self.object.title,
+            'details': self.object.description,
+            'dates': self.object.start_time.strftime('%Y%m%dT%H%M%S') + '/' + end_time.strftime('%Y%m%dT%H%M%S'),
```
Comment (aightmunam) [inline openedx/adg/lms/webinars/views.py:78]: Let's use a constant for `%Y%m%dT%H%M%S`

**Review by aightmunam — CHANGES_REQUESTED** [review]

File: openedx/adg/lms/webinars/views.py (hunk 2 above)
Comment (javeriaejaz-arbisoft) [reply openedx/adg/lms/webinars/views.py:78]: We are not using this format again anywhere else.

Comment (aightmunam) [reply openedx/adg/lms/webinars/views.py:78]: It's being used twice here. We can use a local constant

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3198/workflows/workflows_id/jobs/3198/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/tests/test_views.py (100%)
openedx/adg/lms/webinars/views.py (100%)
-------------
Total:   47 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by faizan-philu — APPROVED** [review]

**Review by aightmunam — APPROVED** [review]
LGTM!

**Pull request merged**